unstructured_inference/inference/layoutelement.py

```diff
@@ -321,13 +327,44 @@ def get_elements_from_ocr_regions(ocr_regions: List[TextRegion]) -> List[LayoutE
             r.y1,
             r.x2,
             r.y2,
+            None,
             text=r.text,
             type="UncategorizedText",
         )
         for r in merged_regions
     ]
 
 
+def probably_contained(
```

Comment:
Doesn't this serve the same purpose as `Rectangle.is_almost_subregion_of`?

Reply:
Probably...let me check if I can use that method

Reply:
It seems like you can use `Rectangle.is_almost_subregion_of`.

From
`nested_current = np.array([probably_contained(e, table) for e in not_tables])`
To
`nested_current = np.array([table.is_almost_subregion_of(e) for e in not_tables])`

Reply:
I think the logic is the opposite one, but I'll give it a try. Thanks both 😄